cardano-testnet/cardano-testnet.cabal

```diff
@@ -57,27 +57,28 @@ library
                       , unordered-containers
 
   hs-source-dirs:       src
-  exposed-modules:      Parsers
-                        Parsers.Babbage
-                        Parsers.Byron
-                        Parsers.Cardano
-                        Parsers.Shelley
-                        Parsers.Version
+  exposed-modules:      Cardano.Testnet
+                        Testnet.Byron
                         Util.Assert
```

Comment:
Let's change `Util.x` to `Testnet.Util.x`. We will need to rename `Testnet.Util` to something else or put the functions/types in a suitable `Testnet.Util.x` module